acceptance/tests/custom_facts/having_multiple_facts_in_one_file.rb

```diff
@@ -0,0 +1,40 @@
+test_name 'C14893: Facter should handle multiple facts in a single file' do
+  tag 'risk:high'
+
+  fact_content = <<-EOM
+Facter.add(:test_fact1) do
+  setcode do
+    "test fact 1"
+  end
+end
```

Comment:
\<rant\>No action required: I realize this is pre-existing code.This style/indentation for here-docs makes my eyes bleed. \</rant\>